### PR TITLE
Show WIP message to avoid confusion in real-time reports.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: analysis/report_templates/default.html [hunk 1]
```diff
@@ -16,7 +16,12 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>FuzzBench: {{ experiment.name }} report</title>
+        <title>
```
Comment (jonathanmetzman) [inline analysis/report_templates/default.html]: I think the warning needs to be much more prominent than "(wip)" or else no one will notice.

Comment (inferno-chromium) [reply analysis/report_templates/default.html]: That is in title of tab. More prominent one is in big title on page, see change below.

Comment (inferno-chromium) [reply analysis/report_templates/default.html]: Screenshot from 2020-03-23 15-04-04

Comment (jonathanmetzman) [reply analysis/report_templates/default.html]: 1. I wouldn't use the abbreviation, write out "work in progress"
2. I don't think it's obvious to people what it means for a report to be in progress so explaining this somewhere would help.

File: analysis/report_templates/default.html [hunk 2]
```diff
@@ -16,7 +16,12 @@
...
+            FuzzBench: {{ experiment.name }} report
+            {% if in_progress %}
+            (running)
```
Comment (inferno-chromium) [inline analysis/report_templates/default.html]: This line is on tab title, so there is no space for that long message. Rest modified as per your suggestion.

**Pull request merged**